azurerm/resource_arm_function_app.go

```diff
@@ -49,7 +49,6 @@ func resourceArmFunctionApp() *schema.Resource {
 			"app_service_plan_id": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
```

Comment:
since this no longer forces a new resource, could we also [update the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/function_app.html.markdown#L96)?

Reply:
@tombuildsstuff my bad. PR now includes the update to documentation. 

Thanks for spotting that.